packages/performance/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
-- [changed] Update the transport mechanism of Fireperf events.
+
+
+# 0.3.0
+- [changed] Updated the transport mechanism of performance events.
```

Comment:
Added an option to control the transport mechanism of performance events.

Reply:
Updated with suggested note.